cluster/manifests/skipper/deployment.yaml

```diff
@@ -193,9 +193,11 @@ spec:
           - "-idle-timeout-server={{ .ConfigItems.skipper_idle_timeout_server }}"
           - "-read-timeout-server={{ .ConfigItems.skipper_read_timeout_server }}"
           - "-write-timeout-server={{ .ConfigItems.skipper_write_timeout_server }}"
+{{ if ne .ConfigItems.skipper_routesrv_enabled "exec" }}
```

Comment:
Maybe we should have one block `{{ if ne .ConfigItems.skipper_routesrv_enabled "exec" }}` and nest `{{ if and (ne .ConfigItems.skipper_edit_route_placeholders "" }}` into it

Reply:
this changes skipper deployment to opt-out, so fine